src/shared/styles/theme.ts

```diff
@@ -305,16 +306,8 @@ export const theme = createTheme({
             },
           },
           '&.MuiChip-colorSecondary': {
-            borderWidth: variables.borderWidth.md,
-            border: `${variables.borderWidth.md} solid ${variables.palette.outline}`,
             color: variables.palette.on_surface_variant,
-            backgroundColor: 'transparent',
-            '&:hover': {
-              backgroundColor: variables.palette.on_surface_variant_alfa8,
-            },
-            a: {
-              color: variables.palette.on_surface_variant,
-            },
+            backgroundColor: variables.palette.inverse_on_surface,
```

Comment:
Thanks for doing this styling! You may be aware already, but a more comprehensive update of the Chip component is going to be captured in [M2-6161](https://mindlogger.atlassian.net/browse/M2-6161).

Did you confirm it's okay to remove the hover state for this color variant? Just noting that the `TagsController` looks like it might use `Chip` in a more interactive manner.

If you are confident we can remove the hover state here, then it would make sense to update this other instance of `Chip` where I manually disabled hover state: [ParticipantSnippet.tsx](https://github.com/ChildMindInstitute/mindlogger-admin/blob/8aee5c3f90428f1bb779ff1723ec3bf6e34c057f/src/modules/Dashboard/components/ParticipantSnippet/ParticipantSnippet.tsx#L35-L40)

Reply:
I spent a bunch of time on this today, and I think I've basically taken care of [M2-6161](https://mindlogger.atlassian.net/browse/M2-6161) in doing so 😅.

In the additional commit I pushed up, I:

- Re-implemented the hover styles, but am only conditionally applying them when the Chip is interactive (more specifically, when a `tabindex=0` attribute it present, which MUI automatically adds when an `onClick` or `onDelete` prop is passed to it.

- Updated my revisions to the theme so that hover states appear correct for each color (I followed existing styles where I could, but no hover states are provided for the revised styles in Figma, so I wound up improvising a little bit and did my best to stay true to the new colors).

- Altered the prop interface for Chips to allow the set of MUI chip props in order to support the "variant" prop. The old "secondary" styles used a grey border, instead of a grey background. The MUI Chip's `variant` can be either `"filled"` or `"outlined"` that is intended to control this behaviour, so opted to disambiguate the existing styles and allow for `color` and `variant` to be set separately. This was my attempt at providing some backwards compatibility with these old styles — none of the new instances of the Chip in Figma appear to _use_ an `"outlined"` variant, and I'm not using it in my implementation, but if for some reason we need to revert to the previous "secondary" Chip styling, it can be opted into on an individual basis by setting that prop.

Chips look like so, now:

<img width="168" alt="Screenshot 2024-04-18 at 6 22 13 PM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1670836/93b76558-5847-47e3-ad5f-ca6e15113f26">

Reply:
That's fantastic @JustinNusca, thanks for doing that! IMO you might as well assign yourself to M2-6161 as well as add it to this PR's description (so that it gets auto-linked to this PR).

I think the only other part of completing M2-6161 would be to make sure the chips under the Account Type column on the [Participants screen](https://www.figma.com/file/6BNx63peNVzrHAF1FoZ78x/Multi-Informant-2024?type=design&node-id=1628-123599&mode=design&t=mywOdvfdkUyr1NOM-4) is fully styled to match Figma, which essentially just means changing the "Pending Invite" chip to "✉︎ Pending" and using the appropriate color, if that's something you wouldn't mind squeezing in?

Reply:
Oh sure! Updated:

![Screenshot 2024-04-19 at 11 16 15 AM](https://github.com/ChildMindInstitute/mindlogger-admin/assets/1670836/2f5d9110-84fe-4a54-842f-a90286c6489f)